shared/react-native/ios/Keybase/AppDelegate.m

```diff
@@ -43,6 +45,19 @@ - (BOOL)addSkipBackupAttributeToItemAtPath:(NSString *) filePathString
   return success;
 }
 
+- (void) createBkgReadableDir:(NSString*) path
```

Comment:
It was easiest to just hit the directory, since Go is going to be creating the actual files in there. Rather than try to guess the names of those files, I just hit the whole thing. 

It will do it on every app startup, but that didn't seem too bad to me.